solutions/usaco-988.mdx

```diff
@@ -0,0 +1,46 @@
+---
+id: usaco-988
+source: USACO Bronze 2020 January
+title: Photoshoot
+author: Ananth Kashyap
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^3)$
```

Comment:
why? N <= 1000 so N^3 = 1e9 operations D:

Reply:
Yeah wait shouldn't this be O(n^2)?

Reply:
> Yeah wait shouldn't this be O(n^2)?
yeah i think so

Reply:
lol is it possible to update and remerge a PR

Reply:
> Yeah wait shouldn't this be O(n^2)?

[his solution is O(n^3) :(](https://github.com/cpinitiative/usaco-guide/pull/1961#discussion_r673667982)

Reply:
Ah rip